jupyter-org-client.el

```diff
@@ -987,6 +990,32 @@ is a stream result. Otherwise return nil."
     (when (looking-at-p "\\(?::[\t ]\\|#\\+END_EXAMPLE\\)")
       (line-end-position (unless (eq (char-after) ?:) 0)))))
 
+(defun jupyter-org--find-mime-types (req-types)
+  "Return the keywords in `jupyter-org-mime-types' that match REQ-TYPES.
+
+If a match is not found, return nil. Try to be intelligent and
+return what the user might intend to use.
+
+REQ-TYPES can be a string such as `plain', `plain html', or
+`text/plain'. The string `text' is translated to `:text/plain'
+and `image' to `:image/png'."
+  (when req-types
+    ;; iterate the user-specified mimetypes looking for symbols that match a
+    ;; symbol in `jupyter-org-mime-types'
+    (remove nil ; remove invalid mimetypes
+            (mapcar (lambda (req-type)
+                      (cond
+                       ((not req-type) nil)
+                       ((string= req-type "text") :text/plain)
+                       ((string= req-type "image") :image/png)
+                       (t (let ((regexp (if (string-match "/" req-type)
```

Comment:
I think it would be better to do `(stringp req-type)` here and then remove the `(not req-type)` clause. `cond` will return nil when none of its clauses succeed anyways.